src/backend/distributed/sql/citus--10.0-3--10.1-1.sql

```diff
@@ -1,5 +1,14 @@
 -- citus--10.0-3--10.1-1
 
+-- add the current database to the distributed objects if not already in there.
+-- this is to reliably propagate some of the alter database commands that might be
+-- supported.
+INSERT INTO citus.pg_dist_object SELECT
+  (SELECT oid FROM pg_class WHERE relname = 'pg_database') AS oid,
```

Comment:
maybe 'pg_catalog.pg_database'::regclass

in theory, there could be multiple tables named pg_database